tests/integration/relationships/nested-relationship-test.js

```diff
@@ -139,7 +139,8 @@ test('Sideloaded nested relationships load correctly', function(assert) {
 
       let middleAgerName = get(middleAger, 'name');
       assert.equal(middleAgerName, 'Middle Ager 1', 'MiddleAger name is there');
-      assert.ok(middleAger.get('kids').includes(kid));
+      // THIS TEST WAS WRONG, DAVID is fixing it on master for relationship payloads
+      // assert.ok(middleAger.get('kids').includes(kid));
```

Comment:
It is unclear why this test is wrong. The payload pushed clearly establishes middleAger kids contains kid, and that kid belongsTo the middleAger. Is the use of `includes` the issue?